sdk/tests/conformance/extensions/ext-sRGB.html

```diff
@@ -336,9 +336,9 @@
     // Draw
     var conversions = [
       [   0,   0 ],
-      [  13,  63 ],
-      [  54, 127 ],
-      [ 133, 191 ],
+      [  13,  13 ],
+      [  54,  54 ],
+      [ 133, 133 ],
       [ 255, 255 ]
```

Comment:
I don't think this is correct.  The target buffer is in sRGB, so after the drawUnitQuad, the buffer's data is in sRGB.  However, ReadPixels is blind to color space, so it reads back the sRGB data.  The original table is correct.